testing/init-tests.js

```diff
@@ -169,6 +170,7 @@ function cleanupTestcase() {
   resetExperimentTogglesForTesting(window);
   resetEvtListenerOptsSupportForTesting();
   cancelTimersForTesting();
+  flushPreactEffects();
```

Comment:
Methinks this should be the first line of this function and not the last line. This way, everything related to test execution will be guaranteed to finish before we do things like restore sandboxes, delete page elements, reset global state, etc.

Reply:
Good idea! Done